webserver/webserver/src/main/java/io/helidon/webserver/ClassPathContentHandler.java

```diff
@@ -53,15 +52,15 @@
 
     @Override
     boolean doHandle(Http.RequestMethod method, Path path, ServerRequest request, ServerResponse response) throws IOException {
-        URL url = classLoader.getResource(path.toString());
+        URL url = classLoader.getResource(path.toString().replace('\\', '/'));
         if (url == null) {
             return false;
         }
 
         // If URL exists then it can be directory and we have to try locate a welcome page
         String welcomePageName = getWelcomePageName();
         if ((welcomePageName != null) && !welcomePageName.isEmpty()) {
-            URL welcomeUrl = classLoader.getResource(path.resolve(welcomePageName).toString());
+            URL welcomeUrl = classLoader.getResource(path.resolve(welcomePageName).toString().replace('\\', '/'));
```

Comment:
I think that issue #256 mentioned above covers this case also.

Reply:
That's very similar to the comment above. However this still using `Path.resolve`.

The welcomePageName is a path relative to the resource root right ? 
Then we could have a slightly different logic, how about something like this:

```java
URL root = classLoader.getResource(resourcePath);
URI root.toUri().resolve(welcomePageName);
```

Reply:
Yes it would work if the welcomePageName file was in the root directory. But if you test it for example on mp1_1-static-content example helidon project, you will see the exact reason why I have done it this way.

Reply:
Looking at the mp1_1-static-content example, the welcome file name configured is `resource.html`, the static content is configured as `/WEB` and `/WEB` is a bundled from `src/main/resources/WEB` which contains `resource.html`.

I.e. welcomeFileName is a path relative to the root. How does it illustrate your point ?

> Yes it would work if the welcomePageName file was in the root directory.

Are you saying it wouldn't work if the welcome file is nested under subdirectories ? The mp1_1-static-content example has it at the first level.

Can you please describe the problem explicitly and explain why your approach works and my suggestion does not.

Reply:
The method URL.toUri().resolve() does not work correctly - when it ends without a slash, it takes the last element of the URI as a resource and not as path, so instead of resolving to ".../classes/WEB/welcome.html" it resolves into ".../classes/welcome.html". If the URL ended with a "WEB/" it would work fine.
As this is to be refactored in #256 , let us keep the workaround.